web/hooks/use-group-supabase.ts

```diff
@@ -238,3 +241,49 @@ export async function setTranslatedMemberRole(
     setRole(null)
   }
 }
+
+export function useRealtimeGroup(groupSlug: string) {
+  const [group, setGroup] = useState<Group | null>(null)
+  function fetchGroup() {
+    getGroupFromSlug(groupSlug)
+      .then((result) => {
+        setGroup(result)
+      })
+      .catch((e) => console.log(e))
+  }
+
+  useEffect(() => {
+    fetchGroup()
+  }, [])
+
+  useEffect(() => {
+    const channel = db.channel('group-realtime')
+    channel.on(
+      'postgres_changes',
+      {
+        event: '*',
+        schema: 'public',
+        table: 'groups',
+        filter: `data->>slug=eq.${groupSlug}`,
+      },
+      (payload) => {
+        fetchGroup()
```

Comment:
I'm pretty sure payload contains the updated group... Did you console.log payload?
Then you would not need to fetch it.